client/src/theme/styles/components/frontend/project/_resource-thumbnail-list.scss

```diff
@@ -12,20 +13,12 @@
     padding-left: 20px;
     margin-bottom: 20px;
 
-    &:nth-last-of-type(-n + 2) {
-      display: none;
-    }
-
     @include respond($break65) {
```

Comment:
Let's drop this breakpoint altogether, so the width goes straight from 50% to 25% at `$break90`.

Reply:
It looks a little weird at 50% right after the breakpoint. Its about a 1:3 aspect ratio.
![Screen Shot 2019-05-02 at 1 42 25 PM](https://user-images.githubusercontent.com/9158517/57105743-6ab17680-6ce0-11e9-89a8-8240529837a1.png)

Reply:
OK, let's set the breakpoint to `$break80` rather than `$break90`.